test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -224,6 +224,68 @@ namespace integration_framework {
     return *this;
   }
 
+  IntegrationTestFramework &IntegrationTestFramework::sendTxSequence(
+      const shared_model::interface::TransactionSequence &tx_sequence,
+      std::function<void(std::vector<shared_model::proto::TransactionResponse>
+                             &)> validation) {
+    log_->info("send transactions");
+    auto transactions = tx_sequence.transactions();
```

Comment:
No need to copy here, maybe use reference?

Reply:
its a test, but ok